compiler/src/openxla/compiler/nvgpu/Transforms/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ iree_cc_library(
     "Passes.h.inc"
   SRCS
     "ConvertHLOToCUDNN.cpp"
+    "NormalizeHLOConvolutionLayouts.cpp"
```

Comment:
These transforms has to be moved to nvgpu/Dialect/CUDNN/Transforms for consistency with IREE. Let's do it as a separate change once you'll be done with changes in flight.

Reply:
Thanks!

Reply:
The pass does not transform cudnn but stablehlo. Are you sure you want it moved?